### PR TITLE
feat: Add bridging to L1 (SC-608)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: test/mainnet-fork/CCTPCalls.t.sol [hunk 1]
```diff
@@ -130,6 +141,50 @@ contract TransferUSDCToCCTPIntegrationTests is ForkTestBase {
         );
     }
 
+}
+
+contract L2ControllerTransferUSDCToCCTPFailureTests is BaseChainUSDCToCCTPTestBase {
```
Comment (hexonaut) [inline test/mainnet-fork/CCTPCalls.t.sol:146]: `Foreign` instead of `L2`.

File: test/mainnet-fork/CCTPCalls.t.sol [hunk 2]
```diff
@@ -130,6 +141,50 @@ contract TransferUSDCToCCTPIntegrationTests is ForkTestBase {
...
+
+    using DomainHelpers for *;
+
+    uint32 constant DOMAIN_ID_CIRCLE_ARBITRUM = 3;
```
Comment (hexonaut) [inline test/mainnet-fork/CCTPCalls.t.sol:150]: Can be removed for forwarder contract.

**Review by hexonaut — CHANGES_REQUESTED** [review]
Minor stuff. Looks good otherwise.

File: test/mainnet-fork/CCTPCalls.t.sol (hunk 2 above)
Comment (lucas-manuel) [reply test/mainnet-fork/CCTPCalls.t.sol:150]: Fixed

**Review by hexonaut — APPROVED** [review]

**Pull request merged**